lib/components/menu_bar/README.md

```diff
@@ -0,0 +1,248 @@
+# ScenicWidgets - MenuBar
+
+This component renders a drop-down menu bar with configurable
+menu contents.
+
+Demo video (5 minutes): https://youtu.be/k1kiCL9oMf4
+
+## How to use the MenuBar component in your Scenic app
+
+This is an example of all the code required to render a MenuBar.
+
+```
+vp_width = 1000         # fetch the viewport width from Scenic
+menu_bar_height = 40    # pick whatever you like
+{:fixed, 220}           # sets how wide the columns of the menus are
+
+# calculate font data
+{:ok, ibm_plex_mono_metrics} =
+    TruetypeMetrics.load("./assets/fonts/IBMPlexMono-Regular.ttf")
```

Comment:
Rather than having the user pass in a font metrics, we can change the menu bar to accept the font name from the Scenic assets

Basically call this code from `{:ok, {_type, font_metrics}} = Scenic.Assets.Static.meta(:roboto)` with the passed in font from the user.